en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -87,7 +88,7 @@ Please connect the 6pin connector of the HW\_PM module to the flight control `Po
 A telemetry system allows you to communicate with, monitor, and control a vehicle in flight from a ground station (for example, you can direct the UAV to a particular position, or upload a new mission).
 
 The communication channel is via Telemetry Radios.
-The vehicle-based radio should be connected to the **TELEM1/TELEM2** port (if connected to this port, no further configuration is required).
+The vehicle-based radio should be connected to the **TELEM1** or **TELEM2** port (if connected to these ports, no further configuration is required).
```

Comment:
@cuhome - FYI only I split this out because otherwise it reads like we have just one port for both telem1 and telem2

Reply:
ok